JitsiConference.js

```diff
@@ -595,6 +595,12 @@ JitsiConference.prototype.unlock = function () {
  * @throws NetworkError or InvalidStateError or Error if the operation fails.
  */
 JitsiConference.prototype.selectParticipant = function(participantId) {
+    // if there is no participants there is no point in sending message
+    // who is on stage, also in this moment datachannel
+    // may be not yet established
+    if (!this.participants || Object.keys(this.participants).length === 0)
+        return;
```

Comment:
The docs say "@throws NetworkError or InvalidStateError or Error if the operation fails".
This code just silently does nothing.

I guess that when this method is called the participantId is either "local" or invalid one and that's what we should verify. Or the docs have to be updated that the method will do nothing in that case.